guides/common/modules/con_composite-content-views-overview.adoc

```diff
@@ -52,4 +52,4 @@ You can also select the *Auto Publish* option when creating a composite content
 
 .Repository restrictions
 Docker repositories cannot be included more than once in a composite content view.
-For example, if you attempt to include two content views using the same docker repository in a composite content view, {ProjectServer} reports an error.
+For example, if you attempt to include two content views by using the same docker repository in a composite content view, {ProjectServer} reports an error.
```

Comment:
alternative:
```suggestion
For example, if you attempt to include two content views with the same docker repository in a composite content view, {ProjectServer} reports an error.
```

or "... that contain ..."

Reply:
```suggestion
For example, if you attempt to include two content views that contain the same docker repository in a composite content view, {ProjectServer} reports an error.
```